packages/styles/src/variables/components/_form-check.scss

```diff
@@ -14,7 +14,27 @@ $form-check-input-size: 1.375rem !default;
 $form-check-input-size-sm: spacing.$size-regular !default;
 $form-check-input-border-width: forms.$input-border-width !default;
 $form-check-input-focus-box-shadow: forms.$input-focus-box-shadow !default;
+$form-check-input-disabled-background-url-light: "data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg'%3e%3crect width='100%25' height='100%25' fill='none' stroke='" +
+  color.$gray-60 + "' stroke-width='" + $form-check-input-border-width
```

Comment:
Very nitpicky, but it looks like the border width is 1px, in Design it's 1.5px for the disabled state and 2px for regular borders. 
![image](https://github.com/swisspost/design-system/assets/1659006/50811bda-9c38-4a4c-bba4-9e886f68a9e5)
Asked for clarification (https://www.figma.com/file/xZ0IW0MJO0vnFicmrHiKaY?node-id=17960:14667&mode=design#742433222)

Reply:
True, I've adjusted the width